src/Resources/translations/messages.en.yml

```diff
@@ -47,3 +47,19 @@ bitbag_sylius_wishlist_plugin:
         wishlist_saved: Wishlist has been saved.
         you_have_no_access_to_that_wishlist: You have no access to that wishlist.
         wishlist_has_product_variant: '%productName% variant is already in wishlist.'
+        wishlist_modal_copy_title: 'Choose a wishlist to which selected items should be copied'
+        wishlist_modal_copy_cancel: 'cancel'
```

Comment:
The point of `bitbag_sylius_wishlist_plugin.ui` namespace is to re-use translations without creating new individual translation for each element.

These should be merged
> wishlist_modal_copy_cancel: 'cancel'
> wishlist_modal_add_cancel: 'cancel'
> wishlist_modal_edit_cancel: 'cancel'
> wishlist_modal_remove_cancel: 'cancel'

or even better, replace it with `sylius.ui.cancel`

Reply:
This is another intention. It's not overridding the translations by setting new "copy" key. Here we've translations for the "copy" functionality. It's not same with `sylius.ui.cancel` context I think.